app/views/profile/show.html.erb

```diff
@@ -10,10 +10,10 @@
 				</ul>
 			</nav>
 -->
-			<img class="uk-thumbnail" src="https://account.socialbakers.com/default_user.png" alt="NÉV">
-			<h3 class="uk-h3"><%= @user.screen_name %></h3>
+			<img class="uk-thumbnail" src="/profile/picture/<%= @user.screen_name %>" alt="<%= @user.screen_name %>">
```

Comment:
Van `image_tag` helper a railsben. Az eleg sok mindent levesz a valladrol az `<img>` tagek renderelesenel. Szerintem itt is lehetne hasznalni.